notebooks/lesson12_nlp-classic.ipynb

```diff
@@ -0,0 +1,1416 @@
+{
```

Comment:
<ul><li>typo in Learning objectives: "python"</li><li>typo in References: "Representing" and i would say "... in Data Science for Business ..."</li><li>suggestion in Notebook overview: "This task is called sentiment analysis and is a ..."</li><li>typo in Notebook overview: "... use a sentiment classifier to analyse ..."</li><li>typo in Notebook overview: "Python offers powerful tools ..."</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lewtun/dslectures/pull/36/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>